pkg/debugapi/welcome_message.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+type welcomeMessageResponse struct {
+	WelcomeMesssage string `json:"welcome_message"`
+}
+
+func (s *server) getWelcomeMessageHandler(w http.ResponseWriter, r *http.Request) {
+	val := s.P2P.GetWelcomeMessage()
+	jsonhttp.OK(w, welcomeMessageResponse{
+		WelcomeMesssage: val,
+	})
+}
+
+func (s *server) setWelcomeMessageHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Given that this is a json-encoded api, using the the body as arbitrary string for the message is not consistent. I would suggest to have welcomeMessageRequest struct to parse and get the message from.

Reply:
Agreed.

Also, I specifically do not check for the max body size, as @janos will create a new PR to add a new middleware specific for this task.